spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -185,6 +197,8 @@ def main(argv: Sequence[str]):
     param.bval_min = arguments.bvalmin
     param.group_size = arguments.g
     param.fname_mask = arguments.m
+    if arguments.ref is not None:
+        param.fname_ref = arguments.ref
```

Comment:
We won't need to check `if ... is not None` anymore:


```suggestion
    param.fname_ref = arguments.ref
```

Reply:
Addressed by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/5100/commits/951078d6b5b87434a6a76c5045a302b85909193a